CHANGES.md

```diff
@@ -1,9 +1,44 @@
-Unreleased
-----------
+3.18.0~alpha0 (2025-03-32)
+--------------------------
 
-If you're a contributor, please include your CHANGES entry in a file
-`doc/changes/$PR_NAME.md`. At release time, it will be incoporated into the
-changelog properly.
+### Fixed
+
+- Support HaikuOS: don't call `execve` since it's not allowed if other pthreads
+  have been created. The fact that Haiku can't call `execve` from other threads
+  than the principal thread of a process (a team in haiku jargon), is a
+  discrepancy to POSIX and hence there is a [bug about
+  it](https://dev.haiku-os.org/ticket/18665). (@Sylvain78, #10953)
+- Fix flag ordering in generated Merlin configurations (#11503, @voodoos, fixes
+  ocaml/merlin#1900, reported by @vouillon)
+- Display negative error codes on Windows in hex which is the more customary
+  way to display `NTSTATUS` codes (#11504, @MisterDA)
+
+### Added
+
+- Add `(format-dune-file <src> <dst>)` action.  (#11166, @nojb)
+- Allow the `--prefix` flag when configuring dune with `ocaml configure.ml`.
+  This allows to set the prefix just like `$ dune install --prefix`. (#11172,
+  @rgrinberg)
+- Allow arguments starting with `+` in preprocessing definitions (starting with
+  `(lang dune 3.18)`). (@amonteiro, #11234)
+- Support for opam `(maintenance_intent ...)` in dune-project (#11274, @art-w)
+- Validate opam `maintenance_intent` (#11308, @art-w)
+- Support `not` in package dependencies constraints (#11404, @art-w, reported
+  by @hannesm)
+
+### Changed
+
+- Format long lists in s-expressions to fill the line instead of formatting
+  them in a vertical way (#10892, fixes #10860, @nojb)
```

Comment:
I think it might be important to state that this only happen in dune-lang 3.18+.

Reply:
If the entry is removed from the 3.18 release, it should be put back for 3.19.